documentation/specs/src/base-ledger/governance.md

```diff
@@ -108,81 +122,97 @@ This means that corresponding VPs need to handle these cases.
+If delegators are allowed to vote, validators will be able to vote only for 2/3 of the total voting period, while delegators can vote until the end of the voting period.
+
+If a delegator votes differently than its validator, this will *override* the corresponding vote of this validator (e.g. if a delegator has a voting power of 200 and votes opposite to the delegator holding these tokens, than 200 will be subtracted from the voting power of the involved validator).
 
 As a small form of space/gas optimization, if a delegator votes accordingly to its validator, the vote will not actually be submitted to the chain. This logic is applied only if the following conditions are satisfied:
 
 - The transaction is not being forced
 - The vote is submitted in the last third of the voting period (the one exclusive to delegators). This second condition is necessary to prevent a validator from changing its vote after a delegator vote has been submitted, effectively stealing the delegator's vote.   
 
 ### Tally
-At the beginning of each new epoch (and only then), in the `FinalizeBlock` event, tallying will occur for all the proposals ending at this epoch (specified via the `grace_epoch` field of the proposal).
-The proposal has a positive outcome if 2/3 of the staked `NAM` total is voting `yay`. Tallying is computed with the following rules:
 
-- Sum all the voting power of validators that voted `yay`
-- For any validator that voted `yay`, subtract the voting power of any delegation that voted `nay`
-- Add voting power for any delegation that voted `yay` (whose corresponding validator didn't vote `yay`)
-- If the aformentioned sum divided by the total voting power is >= `2/3`, the proposal outcome is positive otherwise negative.
+At the beginning of each new epoch (and only then), in the `finalize_block` function, tallying will occur for all the proposals ending at this epoch (specified via the `grace_epoch` field of the proposal).
+The proposal has a positive outcome if the threshold specified by the `ProposalType` is reached. This means that enough `yay` votes must have been collected: the threshold is relative to the staked `NAM` total.
+
+Tallying, when no `memo` is required, is computed with the following rules:
+
+1. Sum all the voting power of validators that voted `yay`
+2. For any validator that voted `yay`, subtract the voting power of any delegation that voted `nay`
+3. Add voting power for any delegation that voted `yay` (whose corresponding validator didn't vote `yay`)
+4. If the aforementioned sum divided by the total voting power is greater or equal to the threshold set by `ProposalType`, the proposal outcome is positive otherwise negative.
 
-All the computation above must be made at the epoch specified in the `start_epoch` field of the proposal.
+If votes carry a `memo`, instead, the `yay` votes must be evaluated net of it. The protocol will implement the correct logic to make sense of these memos and compute the tally correctly:
```

Comment:
Is there a case where we have to evaluate the memo to compute the tally? we should be able to tally just with `yay` or `nay`, without looking the the memo

Reply:
For example, for PGF, don't we need to look at the council in the memo and then compute the tally based on the different memos we received?

Reply:
ah, right!